test/config_partial_keyspace.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
A lot of these have retries, do we expect them to be flaky?

Reply:
I used the same settings as the original vtgate tests.